packages/shell-color-scripts/shell-color-scripts.pacscript

```diff
@@ -0,0 +1,20 @@
+name="shell-color-scripts"
+version="1.0"
+url="https://gitlab.com/dwt1/shell-color-scripts/-/archive/master/shell-color-scripts-master.tar.gz"
+description="A collection of terminal color scripts."
+hash="ffe50c7ca3ee5da0c7110be0763c64b12e8c858131d8e00e3c7954f2c9770a79"
+maintainer="Fahri Korkmaz <fahri.korkmaz@protonmail.ch>"
+
+prepare() {
+        true
+}
+
+build() {
+        true
+}
+
+install() {
+        sudo install -D colorscript.sh -t "$STOWDIR/$name/usr/bin/"
+        sudo mv "$STOWDIR/$name/usr/bin/colorscript.sh" "$STOWDIR/$name/usr/bin/colorscript"
```

Comment:
Umm.... Why are you moving the same file to the same place?

Reply:
Because the first command will create it as `colorscript.sh`. The second command will rename it to `colorscript`. If I run `sudo install -D colorscript.sh -t "$STOWDIR/$name/usr/bin/colorscript"` the script will place it to `/usr/bin/colorscript/colorscript.sh`. Any suggestions how I can place it directly to `/usr/bin/colorscript`?

Reply:
Yeah, don't use the `-t` flag

```bash
sudo install -D colorscript.sh "$STOWDIR/$name/usr/bin/colorscript"

Reply:
Ok thank you!